anta/tests/flow_tracking.py

```diff
@@ -51,36 +27,49 @@ def validate_exporters(exporters: list[dict[str, str]], tracker_info: dict[str,
 
     Returns
     -------
-    str
+    list
         Failure message if any exporter configuration does not match.
     """
-    failed_log = ""
+    failed_log = []
     for exporter in exporters:
-        exporter_name = exporter["name"]
+        exporter_name = exporter.name
         actual_exporter_info = tracker_info["exporters"].get(exporter_name)
         if not actual_exporter_info:
-            failed_log += f"\nExporter `{exporter_name}` is not configured."
+            failed_log.append(f"Exporter: {exporter_name} - Not configured")
             continue
+        local_interface = actual_exporter_info["localIntf"]
+        template_interval = actual_exporter_info["templateInterval"]
 
-        expected_exporter_data = {"local interface": exporter["local_interface"], "template interval": exporter["template_interval"]}
-        actual_exporter_data = {"local interface": actual_exporter_info["localIntf"], "template interval": actual_exporter_info["templateInterval"]}
+        if local_interface != exporter.local_interface:
+            failed_log.append(f"Exporter: {exporter_name} - Incorrect local interface - Expected: {exporter.local_interface} Actual: {local_interface}")
 
-        if expected_exporter_data != actual_exporter_data:
-            failed_msg = get_failed_logs(expected_exporter_data, actual_exporter_data)
-            failed_log += f"\nExporter `{exporter_name}`: {failed_msg}"
+        if template_interval != exporter.template_interval:
+            failed_log.append(f"Exporter: {exporter_name} - Incorrect template interval - Expected: {exporter.template_interval} Actual: {template_interval}")
     return failed_log
 
 
 class VerifyHardwareFlowTrackerStatus(AntaTest):
-    """Verifies if hardware flow tracking is running and an input tracker is active.
+    """Verifies the hardware flow tracking.
 
-    This test optionally verifies the tracker interval/timeout and exporter configuration.
+    This test performs the following checks:
+
+      1. Confirms that hardware flow tracking is running.
+      2. Confirms that the specified input tracker is active.
+      3. Optionally, checks the tracker interval/timeout configuration.
+      4. Optionally, verifies the tracker exporter configuration
 
     Expected Results
     ----------------
-    * Success: The test will pass if hardware flow tracking is running and an input tracker is active.
-    * Failure: The test will fail if hardware flow tracking is not running, an input tracker is not active,
-               or the tracker interval/timeout and exporter configuration does not match the expected values.
+    * Success: The test will pass if all of the following conditions are met:
+        - All Hardware flow tracking is running.
```

Comment:
Not sure what this means
```suggestion
        - All Hardware flow tracking is running.
```

Reply:
Checking weather the hardware flow tracking is running or not on device

Reply:
you need to reword `Hardware flow tracking is running` - remove the `All`